.github/workflows/ci.yml

```diff
@@ -117,42 +117,24 @@ jobs:
       #
       # The other binaries (like `rustc`) will be overwritten by rustup.
       #
-      # Current size as of 2019-12-23: ~6 MB
+      # Current size as of 2021-02-15: ~15 MB
       - name: Cache cargo binaries
-        uses: actions/cache@v2
+        uses: actions/cache@main
```

Comment:
what's the benefit of running `main`? it looks like the README of https://github.com/actions/cache recommends using `v2`.

Reply:
`v2.1.4` has more verbose output, which I think will be helpful while we evaluate cache performance. For some reason, the `v2` tag still points to the same commit as `v2.1.3` from early November. We could use `v2.1.4` instead of `main`, but in either case I agree we'll want to move back to `v2` eventually.

Reply:
huh, interesting. I guess I'd prefer to use `v2.1.4` then instead of the moving `main` target.

Reply:
Okay, I've added a commit switching to `v2.1.4`.